src/Playroom/SettingsPanel/SettingsPanel.css.ts

```diff
@@ -98,3 +98,25 @@ export const label = style([
     },
   },
 ]);
+
+export const textField = style([
+  sprinkles({
+    font: 'large',
+    width: 'full',
+    paddingX: 'large',
+    boxSizing: 'border-box',
+    borderRadius: 'medium',
+  }),
+  {
+    color: colorPaletteVars.foreground.neutral,
+    height: vars.touchableSize,
+    background: colorPaletteVars.background.surface,
+    ':focus': {
+      outline: 'none',
+    },
```

Comment:
Any particular reason to disable the focus outline?

Reply:
I think this might be a leftover property from before we put it in a box and maybe should be removed now. @benjervis should be able to provide more context.

Reply:
Yes this was the case, I'll remove it. Nice catch!

Reply:
I think this was covering for the implementation of my original mediocre design, but with the way it looks now I think the focus outline is fine and we can put it back